pkg/logs/status/status.go

```diff
@@ -137,3 +139,12 @@ func toDictionary(c *config.LogsConfig) map[string]interface{} {
 	}
 	return dictionary
 }
+
+func init() {
```

Comment:
go-noob, wondering whether those functions always return the value of the init. Let's write some tests to assert that this works properly.

Reply:
i'm not sure what you mean here ? They do not return anything
More docs at https://golang.org/doc/effective_go.html#init